sanity-studio/schemas/yoga-class.js

```diff
@@ -35,5 +43,11 @@ export default {
       type: 'number',
       validation: (Rule) => Rule.min(0),
     },
+    {
+      name: 'order',
+      title: 'Order',
+      type: 'number',
+      hidden: true,
+    },
```

Comment:
What does `order` stands for here? Is it about order of yoga class cards that the admin can edit?

Reply:
yes, correct!